### DISCUSSION TITLE
How to handle checkout sessions?

### DISCUSSION THREAD

**Discussion by aacoimbra** [flutter-stripe/flutter_stripe#1761] [Q&A]
I was wondering how to handle checkout sessions with this sdk.

I've read that this packaged dropped support for checkout sessions in favor of payment links, but I don't see how they are interchangeable. Checkout sessions are nicely embedded in the app with items details and carry user information. 

Also I would prefer not calculating the due amount and let stripe calculate it automatically when my server create a new checkout sessions.

What are the workarounds?

**Comment by jonasbark** [comment]
you're free to continue handling Stripe checkout links in your own app - it doesn't need our plugin to do so

**Reply by aacoimbra** [reply]
Thanks for the reply. 

Could you elaborate a little on how the checkout link should be handled? It's recommended to use web view?
And what do you mean by checkout link exactly?

Currently our back creates a checkout session object and sends the information to the app.

**Reply by aacoimbra** [reply]
@jonasbark I assume embedded mode for checkout is not possible in flutter, am I right?

**Reply by meladlinutech** [reply]
@aacoimbra did you manage to handle checkout sessions in flutter? or did you go in a different way?
